app/refactor/viewmodels/p5/tad/Table2ViewModel.scala

```diff
@@ -47,7 +47,12 @@ object Table2ViewModel {
         .map(
           seal => s"${seal.sequenceNumber}/${seal.identifier}"
         )
-        .firstAndLast(";"),
+        .firstAndLast(";") + "\n" + ie029.Consignment.TransportEquipment
+        .flatMap(_.GoodsReference)
+        .map(
+          goodsReference => s"${goodsReference.sequenceNumber},${goodsReference.declarationGoodsItemNumber}"
+        )
+        .take3(_.semiColonSeparate),
```

Comment:
@malaw-hmrc could we look to add goods references as a separate param in the view model? And handle rendering the seals and goods references on separate lines in the markup?

Reply:
Done, moved them as separate param and updated the markup